app/External/Slack/Modals/EquipmentAuthorization.php

```diff
@@ -96,26 +96,19 @@ public static function handle(SlackRequest $request)
         $selectedEquipment = self::equipmentFromState($state);
         $selectedMembers = self::peopleFromState($state);
         
-        $actorId = $request->customer()->id;
-
-        foreach($selectedMembers->crossJoin($selectedEquipment)->all() as [$person, $equipment]) {
-            if (!$person->hasMembership($equipment->user_plan_id)) {
-                Log::info('EquipmentAuthorization: Customer '.$actorId.' authorized Customer '.$person->id.' to use equipment under plan id '.$equipment->user_plan_id);
-                $api->members->addMembership($person->id, $equipment->user_plan_id);
```

Comment:
I think the code is fine, and you could have just done this one line change here:
```php
app(AuthorizeEquipmentAction::class)->onQueue()->execute($actorId, $person->id, $equipment->user_plan_id);
```